### PR TITLE
GCE: forwarding rules to use generated code

### PR DESCRIPTION
**Pull request by bowei**
GCE: forwarding rules to use generated code

```release-note
NONE
```

### CODE REVIEW

**Review by MrHohn — APPROVED** [review]
/lgtm
/approve no-issue

File: pkg/cloudprovider/providers/gce/gce_forwardingrule.go [hunk 1]
```diff
@@ -31,115 +35,83 @@ func newForwardingRuleMetricContextWithVersion(request, region, version string)
...
-		return mc.Observe(err)
-	}
-
-	return gce.waitForGlobalOp(op, mc)
+	target := &compute.TargetReference{Target: targetProxyLink}
+	return mc.Observe(gce.c.GlobalForwardingRules().SetTarget(context.Background(), meta.GlobalKey(forwardingRuleName), target))
 }
 
-// DeleteGlobalForwardingRule deletes the GlobalForwardingRule by name.
+// DeleteGlobalForwardingRule deletes the GlobalForwakdingRule by name.
```
Comment (yujuhong) [inline pkg/cloudprovider/providers/gce/gce_forwardingrule.go]: s/GlobalForwakdingRule/GlobalForwardingRule

**Pull request merged**